api/app/Console/Kernel.php

```diff
@@ -23,9 +24,15 @@ protected function schedule(Schedule $schedule)
         // clean up old user generated files every day at 2:00 AM
         $schedule->command(PruneUserGeneratedFiles::class)
             ->timezone('America/Toronto')
-            ->dailyAt('2:00')
+            ->dailyAt('1:00')
             ->withoutOverlapping()
             ->appendOutputTo('/tmp/laravel-prune-user-generated-files.log');
+
+        // queue up Application Deadline Approaching emails every day, close to the time the pool would close
+        $schedule->command(SendNotificationsApplicationDeadlineApproaching::class)
```

Comment:
I ended up scheduling this daily instead of hourly like the issue described.  Daily is enough to reliably run this without missing any pools.  Also, if this runs hourly and is slightly off, you risk notifying a pool twice or not at all.

Reply:
Gray confirmed that near the end-of-day in Newfoundland (when it is the same day across the country) is a good time to send the email.